docs/source/howto/queryable-encryption.rst

```diff
@@ -124,12 +124,12 @@ Django settings::
     from bson.binary import Binary
     from pymongo.encryption_options import AutoEncryptionOpts
 
-    …
+
     DATABASES["encrypted"] = {
-        …
+        ...
         "OPTIONS": {
             "auto_encryption_opts": AutoEncryptionOpts(
-                …
+                ...
```

Comment:
@aclark4life fyi: use three dots rather than the ellipses character. The entire code block won't be highlighted if the syntax is invalid, and the ellipsis character isn't valid.